lib/Validations.php

```diff
@@ -732,7 +732,7 @@ public function add_on_blank($attribute, $msg)
 		if (!$msg)
 			$msg = self::$DEFAULT_ERROR_MESSAGES['blank'];
 
-		if (($value = $this->model->$attribute) === '' || $value === null)
+		if (($value = $this->model->read_attribute($attribute)) === '' || $value === null)
```

Comment:
Not a barrier to merging, but how do you feel about moving the assignments outside of `if` statements for lines we touch? I am not a big fan of mixing the assignment and comparison operations.